assets/js/util/tracking/createTracking.js

```diff
@@ -6,7 +6,7 @@ import createEnableTracking from './createEnableTracking';
 import createTrackEvent from './createTrackEvent';
 
 const DEFAULT_CONFIG = {
-	isFirstAdmin: false,
+	isOwner: false,
```

Comment:
See above, this should probably remain `isFirstAdmin`.

Reply:
Updated.